app/Stores/EditStore.js

```diff
@@ -0,0 +1,195 @@
+import { EventEmitter } from 'events';
+import { Map } from 'immutable';
+import objectAssign from 'object-assign';
+
+import AppDispatcher from '../Services/AppDispatcher';
+
+import ReadQueries from 'admin-config/lib/Queries/ReadQueries';
+import WriteQueries from 'admin-config/lib/Queries/WriteQueries';
+import PromisesResolver from 'admin-config/lib/Utils/PromisesResolver';
+import DataStore from 'admin-config/lib/DataStore/DataStore';
+
+import RestWrapper from '../Services/RestWrapper';
+
+class EditStore extends EventEmitter {
```

Comment:
Why don't we have only one application store? I don't understand the rationale to structure the model (the store) according to the view.